bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -103,6 +103,18 @@
 			</options>
 		</state>
 	</channel-type>
+	<channel-type id="audysseyeq" advanced="true">
+		<item-type>Number</item-type>
+		<label>Audyssey EQ Mode</label>
+		<description>Choose between Audyssey 2EQ/MultiEQ Equalizer curves</description>
+		<state>
+			<options>
+				<option value="0">Audyssey EQ Off</option>
+				<option value="1">Audyssey EQ Movie</option>
+				<option value="2">Audyssey EQ Music</option>
```

Comment:
```suggestion
				<option value="0">Off</option>
				<option value="1">Movie</option>
				<option value="2">Music</option>
```

As the channel and label is allready Audyssey EQ, this duplicate info is aonly cluttering the ui. Please remove and also re-run the i18n plugin as you did before to also update the translation files.

Reply:
done